repos/system_upgrade/el7toel8/actors/peseventsscanner/libraries/peseventsscanner.py

```diff
@@ -475,44 +478,55 @@ def map_repositories(packages):
 
     if repo_without_mapping:
         report_skipped_packages('packages will not be installed or upgraded due to repositories unknown to leapp:',
-                                repo_without_mapping)
+                                repo_without_mapping,
+                                "Please file a bug in http://bugzilla.redhat.com/ for leapp-repository component of"
+                                "the Red Hat Enterprise Linux 7 product.")
 
 
-def report_skipped_packages(message, packages):
+def report_skipped_packages(message, packages, remediation=None):
     """Generate report message about skipped packages"""
     packages = sorted(packages)
     title = 'Packages will not be installed'
     summary = '{} {}\n{}'.format(len(packages), message, '\n'.join(['- ' + p for p in packages]))
-    reporting.create_report([
+    report_content = [
         reporting.Title(title),
         reporting.Summary(summary),
         reporting.Severity(reporting.Severity.HIGH),
         reporting.Tags([reporting.Tags.REPOSITORY]),
-    ] + [reporting.RelatedResource('package', p) for p in packages])
+    ]
+    if remediation:
+        report_content += [reporting.Remediation(hint=remediation)]
+    report_content += [reporting.RelatedResource('package', p) for p in packages]
+    reporting.create_report(report_content)
     if is_verbose():
-        api.show_message(summary)
+        api.current_logger().info(summary)
 
 
 def add_output_pkgs_to_transaction_conf(transaction_configuration, events):
     """
-    Add more packages for removal to transaction configuration if they can be derived as outputs of PES events.
+    Filter out those PES events that conflict with the higher priority transaction configuration files.
 
     Output packages from an event are added to packages for removal only if all input packages are already there.
 
     :param transaction_configuration: RpmTransactionTasks model instance with pkgs to install, keep and remove based
                                       on the user configuration files
     :param events: List of Event tuples, where each event contains event type and input/output pkgs
     """
-    message = 'Marking packages for removal:\n'
+    message = 'The following RHEL 8 packages will not be installed:\n'
```

Comment:
Mentioning the target distro by name means the text need to become dynamic once we spit off to el8toel9.

Reply:
Not so big problem. It's highly possible the actor will have to be rewritten because of modularity anyway in the future.